tests/parsers/test.manifestjson.js

```diff
@@ -25,6 +25,40 @@ describe('ManifestJSONParser', function() {
 
 });
 
+describe('ManifestJSONParser id', function() {
+
+  it('should return the correct id', () => {
+    var json = validManifestJSON();
+    var manifestJSONParser = new ManifestJSONParser(json);
+    assert.equal(manifestJSONParser.isValid, true);
+    var metadata = manifestJSONParser.getMetadata();
+    assert.equal(metadata.id, '{daf44bf7-a45e-4450-979c-91cf07434c3d}');
+  });
+
+  it('should barf on invalid id', () => {
```

Comment:
This is a funny message but might be a bit inaccessible to ESLers, just saying.

ANTI-WHISMY 2016! 😅